libs/react-helpers/src/lib/constants/links.ts

```diff
@@ -0,0 +1,32 @@
+/** ExternalLinks returns external documentation links that are specific to an environment
```

Comment:
chore (blocking): Can we make this file just `lib/links`, or something similar? Because of the DocsLinks function the values here are no longer just constants.

Reply:
done. updated in last commit.